Domains/4-Application/ISM/Released/IntegratedStructuralModel.02.00.00.ecschema.xml

```diff
@@ -0,0 +1,2764 @@
+        <ECEnumerator value="20" name="Round" displayLabel="Round" />
+        <ECEnumerator value="30" name="Oval" displayLabel="Oval" />
+    </ECEnumeration>
+
+    <ECEnumeration typeName="DuctMaterial" backingTypeName="int" isStrict="true">
+        <ECEnumerator value="0" name="Unspecified" displayLabel="Unspecified" />
+        <ECEnumerator value="10" name="Plastic" displayLabel="Plastic" />
+        <ECEnumerator value="20" name="Steel" displayLabel="Steel" />
+    </ECEnumeration>
+
+    <ECEnumeration typeName="PostTensioningSystem" backingTypeName="int" isStrict="true">
+        <ECEnumerator value="0" name="Unspecified" displayLabel="Unspecified" />
+        <ECEnumerator value="10" name="Bonded" displayLabel="Bonded" />
+        <ECEnumerator value="20" name="Unbonded" displayLabel="Unbonded" />
+    </ECEnumeration>
+
+    <ECEnumeration typeName="AnchorType" backingTypeName="int" isStrict="true">
+        <ECEnumerator value="0" name="Unspecified" displayLabel="Unspecified" />
+        <ECEnumerator value="10" name="Monostrand" displayLabel="Monostrand" />
+        <ECEnumerator value="20" name="FlatMultiPlane" displayLabel="Flat Multi Plane" />
+        <ECEnumerator value="30" name="FlatSinglePlane" displayLabel="Flat Single Plane" />
+        <ECEnumerator value="40" name="CircularMultiPlane" displayLabel="Circular Multi Plane" />
+        <ECEnumerator value="50" name="CircularSinglePlane" displayLabel="Circular Single Plane" />
+        <ECEnumerator value="60" name="SquareMultiPlane" displayLabel="Square Multi Plane" />
+        <ECEnumerator value="70" name="SquareSinglePlane" displayLabel="Square Single Plane" />
+    </ECEnumeration>
+
+    <ECEntityClass typeName="IsmTendonSet" modifier="None" displayLabel="Tendon Set">
+        <BaseClass>bis:InformationRecordElement</BaseClass>
+        <BaseClass>IIsmObject</BaseClass>
+    </ECEntityClass>
+
+    <ECRelationshipClass typeName="IsmTendonSet_Members" modifier="None" strength="referencing" strengthDirection="forward">
+        <BaseClass>IsmObject_Object</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="is located in members" polymorphic="false">
+            <Class class="IsmTendonSet" />
+        </Source>
+        <Target multiplicity="(0..*)" roleLabel="has tendon sets" polymorphic="true">
+            <Class class="IsmSpanningMember" />
+        </Target>
+        <ECCustomAttributes>
+            <RelationshipHasLabels>
+                <SourceDisplayLabel>Tendon sets</SourceDisplayLabel>
+                <TargetDisplayLabel>Members</TargetDisplayLabel>
+            </RelationshipHasLabels>
+        </ECCustomAttributes>
+    </ECRelationshipClass>
+
+    <PropertyCategory typeName="IsmTendonNode_category" displayLabel="Tendon Node" priority="190" />
+    <ECEntityClass typeName="IsmTendonNode" modifier="None" displayLabel="Tendon Node">
+        <BaseClass>bis:PhysicalElement</BaseClass>
```

Comment:
Does IsmTendonNode model "the physical element"? Or is it IsmTendon?

Reply:
Changed to SpatialLocationElement